<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской Бой</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                    stop:0 #0a2463, stop:1 #3e92cc);
    border: 3px solid #1e3d6b;
}

QLabel {
    color: #f8f8ff;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.5);
}

QLabel#labelTitle {
    font-size: 48px;
    font-weight: bold;
    color: #f8f8ff;
    background-color: rgba(30, 61, 107, 0.7);
    border-radius: 15px;
    padding: 10px;
    border: 2px solid #d4af37;
}

QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #1e3d6b, stop:1 #0a2463);
    border: 2px solid #d4af37;
    border-radius: 15px;
    padding: 12px 24px;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    font-size: 18px;
    color: #f8f8ff;
    min-width: 200px;
    min-height: 50px;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #3a5f8c, stop:1 #1e3d6b);
    border: 2px solid #f8f8ff;
}

QPushButton:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #0a2463, stop:1 #1e3d6b);
}

QPushButton#btnExitToDesktop {
    min-width: 200px;
    min-height: 50px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #c62828, stop:1 #b71c1c);
    border: 2px solid #e57373;
}

QPushButton#btnExitToDesktop:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #ef5350, stop:1 #e53935);
}

QPushButton#btnExitToDesktop:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #b71c1c, stop:1 #7f0000);
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>МОРСКОЙ БОЙ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>361</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Игра против бота</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Игра с другом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExitToDesktop">
       <property name="text">
        <string>Выход на рабочий стол</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
